internal/engine/ingester/artifact/artifact.go

```diff
@@ -36,13 +44,39 @@ const (
 // Ingest is the engine for a rule type that uses artifact data ingest
 // Implements enginer.ingester.Ingester
 type Ingest struct {
+	ghCli provifv1.GitHub
+
+	testOverrideVerifier *verifier.Verifier
+}
+
+type verification struct {
+	IsSigned          bool   `json:"is_signed"`
+	IsVerified        bool   `json:"is_verified"`
+	Repository        string `json:"repository"`
+	Branch            string `json:"branch"`
+	WorkflowName      string `json:"workflow_name"`
+	RunnerEnvironment string `json:"runner_environment"`
+	CertIssuer        string `json:"cert_issuer"`
+}
+
+type verificationResult struct {
+	Verification verification `json:"Verification"`
```

Comment:
Is the JSON capitalization intentional?

Reply:
It's because what is already used in the rego rule type where we loop through its entries. If we change it we'll have to update the ruletype to match and I suppose this will break previously deployed rule types where their rego expression will be expecting it to be capitalized.

Reply:
Scratch that. I was wrong we rely on parsing this from the eval.